src/index.ts

```diff
@@ -21,7 +21,11 @@ type Experience = {
 }
 
 type Placement = {
-    content: object,
+    url: string,
```

Comment:
It is weird, that Placement returned by Android SDK in Java/Kotlin is completely not typed, content can be everything, while in the Javascript/Typescript wrapper it becomes strongly typed.
Are you sure Placement has always these properties: url, message, image?

Reply:
Yes, my bad. Forgot that placement content can be anything. Reverted.

Reply:
But reverting previous version, we return to the previous problem: returned object doesn't have `content` property.